frontend/insurance/src/sass/spacing/spacing.scss

```diff
@@ -1,72 +1,79 @@
 .mls {
-    margin-left: 4px !important;
-  }
-  .mlm {
-    margin-left: 8px !important;
-  }
-  .mll {
-    margin-left: 16px !important;
-  }
-  .mlh {
-    margin-left: 32px !important;
-  }
-  
-  .mtn {
-    margin-top: 0px !important;
-  }
-  .mts {
-    margin-top: 4px !important;
-  }
-  .mtm {
-    margin-top: 8px !important;
-  }
-  .mtl {
-    margin-top: 16px !important;
-  }
-  .mth {
-    margin-top: 32px !important;
-  }
-  
-  .mrs {
-    margin-right: 4px !important;
-  }
-  .mrm {
-    margin-right: 8px !important;
-  }
-  .mrl {
-    margin-right: 16px !important;
-  }
-  
-  .mbs {
-    margin-bottom: 4px !important;
-  }
-  .mbm {
-    margin-bottom: 8px !important;
-  }
-  .mbl {
-    margin-bottom: 16px !important;
-  }
-  .mbh {
-    margin-bottom: 32px !important;
-  }
-  
-  .pls {
-    padding-left: 4px !important;
-  }
-  
-  .pbs {
-    padding-bottom: 4px !important;
-  }
-  
-  .pbm {
-    padding-bottom: 8px !important;
-  }
-  
-  .pbl {
-    padding-bottom: 16px !important;
-  }
-  
-  .pbh {
-    padding-bottom: 32px !important;
-  }
-  
\ No newline at end of file
+  margin-left: 4px !important;
```

Comment:
What happened here and why did you change the class name? Did you fix all references to this class?